frontend/src/components/SocialMediaButton/index.js

```diff
@@ -9,60 +10,35 @@ import Dribbble from "../../assets/imgs/socialButtons/dribble.png";
 import Medium from "../../assets/imgs/socialButtons/medium.png";
 import Twitter from "../../assets/imgs/socialButtons/twitter.png";
 
-const socialMediaButton = (props) => {
-  const name = props.name;
-  const username = props.username;
+const socialMediaButton = ({ name, username }) => {
+  let i;
 
-  let logo = "";
-  let link = "";
-  const links = {
-    github: "https://github.com/",
-    linkedin: "https://www.linkedin.com/company/", // if its an user: "https://www.linkedin.com/in/",
-    behance: "https://www.behance.net/",
-    facebook: "https://www.facebook.com/",
-    instagram: "https://www.instagram.com/",
-    dribbble: "https://dribbble.com/",
-    medium: "https://medium.com/",
-    twitter: "https://twitter.com/",
-  };
-
-  if (name === "github") {
-    logo = Github;
-    link = links.github + username;
-  }
-  if (name === "linkedin") {
-    logo = LinkedIn;
-    link = links.linkedin + username;
-  }
-  if (name === "behance") {
-    logo = Behance;
-    link = links.behance + username;
-  }
-  if (name === "facebook") {
-    logo = Facebook;
-    link = links.facebook + username;
-  }
-  if (name === "instagram") {
-    logo = Instagram;
-    link = links.instagram + username;
-  }
-  if (name === "dribbble") {
-    logo = Dribbble;
-    link = links.dribbble + username;
-  }
-  if (name === "medium") {
-    logo = Medium;
-    link = links.medium + username;
-  }
-  if (name === "twitter") {
-    logo = Twitter;
-    link = links.twitter + username;
+  const info = {
```

Comment:
I suggest thinking of a better name than `info`. Try to be descriptive.

Reply:
Could you use the social networks name as a key and have the `url` as value?

Reply:
```
const socialMedias = {
    "": { url: "#", logo: Default },
    "github": { url: `https://github.com/${username}`, logo: Github },
    "linkedin": { url: `https://www.linkedin.com/company/${username}`, logo: LinkedIn },
    ...
  }
```
Should it be something like this?

Reply:
I don't think `"": { url: "#", logo: Default },` should be part of this object. If needed, you can return that in an `else` in the function that you're still to create.

Reply:
Also,  you could also use the key for `logo`. That way, you wouldn't need `url` and `logo` but just `"github": https://github.com/${username}` - right?